crates/ruff/src/rules/ruff/rules/pairwise_over_zipped.rs

```diff
@@ -93,7 +93,7 @@ pub fn pairwise_over_zipped(checker: &mut Checker, func: &Expr, args: &[Expr]) {
         return;
     };
 
-    if !(args.len() > 1 && id == "zip" && checker.ctx.is_builtin(id)) {
+    if !(args.len() == 2 && id == "zip" && checker.ctx.is_builtin(id)) {
```

Comment:
I think we may want to avoid _both_ `args != 2` _and_ `strict=True`, since the latter isn't supported?

Reply:
I think this current approach only catches extra arguments, but not the unsupported kwargs.

Reply:
Done.

Reply:
@charliermarsh  But isn't this a valid case for pairwise `zip(foo[:-1], foo[1:], strict=True)`?

Reply:
Indeed

Reply:
Oh hmm, in that case, isn't _either_ `strict` argument acceptable? (`False` is of course fine.)